### ISSUE TITLE
Pywal Support

### ISSUE THREAD

**Issue by RandomLegend** [adi1090x/rofi#103]
Hey adi!

first of all, thanks for your great designs!

Are there any plans to implement pywal support for those rofi themes here? You got those for polybar and pywal itself has support for rofi.
I just can't figure out how to automate this with your templates here. Especially with application launcher type 6, where the colors are hardcoded. Would be awesome to have this automated as the rest aswell.

Thanks for any answer! :-)

**Comment by adi1090x** [comment]
Currently not implemented, I'll try to do that when I've some free time.

**Comment by RandomLegend** [comment]
Yeah i just built a script to do that out of some parts of other pywal "integration" scripts.

I never REALLY used Github. Should i make like a pullrequest with that script?
